test/test-async.js

```diff
@@ -2806,20 +2999,20 @@ exports['cargo bulk task'] = function (test) {
 };
 
 exports['cargo drain once'] = function (test) {
-   
+
```

Comment:
These are trailing whitespace, no lines deleted.  My editor automatically does that.